framework/source/class/qx/ui/embed/Iframe.js

```diff
@@ -335,7 +380,11 @@ qx.Class.define("qx.ui.embed.Iframe",
 
       if (iframeSource != this.getSource())
       {
-        qx.bom.Iframe.getWindow(iframeDomElement).stop();
+        if ( qx.core.Environment.get("browser.name") != "edge" &&
+             qx.core.Environment.get("browser.name") != "ie" )
+        {
+          qx.bom.Iframe.getWindow(iframeDomElement).stop();
+        }
```

Comment:
@schnepe2 why is this specific to ie and edge and was not before?
Ist this something specific to the mutation even?